### PR TITLE
[RFR] Create a deploy gh-pages task

### PR DESCRIPTION
**Pull request by jpetitcolas**
With some automatic commit inside. But it renders a nice [GitHub page](http://marmelab.com/EventDrops/) now.

Updating the README with this link will come later: it needs more fancy styling first.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
+demo/eventDrops.js
```
Comment (jpetitcolas) [inline .gitignore]: Not used

**Comment by fzaninotto** [comment]
:+1:

**Comment by fzaninotto** [comment]
tests fail

**Comment by jpetitcolas** [comment]
Just added better cleaning of demo folder after a deployment and fixed tests.

**Pull request merged**